go.mod

```diff
@@ -0,0 +1,20 @@
+module github.com/mattermost/mattermost-plugin-github
+
+go 1.12
+
+require (
+	github.com/go-ldap/ldap v3.0.3+incompatible // indirect
+	github.com/google/go-github v17.0.0+incompatible
```

Comment:
Maybe this is a good time to update this? v17.0.0 is a year old.

Reply:
If we update that we need to change some code there are some breaking changes

Reply:
However if need i can do the update

Reply:
I'm 1/5 that this dependency should be updated at some point. 0/5 if this PR is the right place.

Reply:
Another PR.